ansible/roles/products/common/resources/templates/scripts/server-setup.j2

```diff
@@ -0,0 +1,16 @@
+#jinja2: trim_blocks:False
+#! /bin/bash
+# TODO: FIX so that cyberglove is not mentioned here in docker-container role, and cyberglove setup is handled in cyberglove role
+if {{ use_cyberglove | lower or remote_cyberglove | lower }}; then
+    if [ -d "/home/user/projects/shadow_robot/base/src/sr_config/sr_cyberglove_config" ]; then
+        rm -r /home/user/projects/shadow_robot/base/src/sr_config/sr_cyberglove_config
+    fi
+    cd /home/user/projects/shadow_robot/base/src
+    if [ -d "sr_cyberglove_config" ]; then
+        cd sr_cyberglove_config
+    else
+        git clone https://github.com/shadow-robot/sr_cyberglove_config.git;
+        cd sr_cyberglove_config
+    fi
+    git checkout {{ cyberglove }}
+fi
```

Comment:
this is to change cyberglove  branch on the server right? It shouldnt be here as well

Reply:
turned out we weren't using server-setup.j2 anywhere, so I deleted it